trifid/redirect.js

```diff
@@ -53,7 +53,10 @@ function factory () {
         } else {
           versionPath = tags[0].name
         }
+      } else if (ref) {
+        versionPath = ref
       }
```

Comment:
Let's handle the case where a version is defined:

```suggestion
      } else if (version) {
        versionPath = version
      }
```